### DISCUSSION TITLE
Expose RUNNER_NAME as environment variables

### DISCUSSION THREAD

**Discussion by soulseekah** [community/community#137836] [Actions] [answered]
### Select Topic Area

Question/Feature Request

### Body

The `RUNNER_NAME` environment variable seems to be missing within actions. We currently pass it as an environment variable from via `${{ runner.name }}` but it seem to be logical to have it passed in by default like the `RUNNER_OS`, `RUNNER_ARCH` variables. Are we missing something perhaps?

**Comment by armyost** [comment, marked as answer]
When I look into this doc, the `RUNNER_NAME`  `RUNNER_OS`  looks default variables. Did you describe like `${{ env.RUNNER_NAME }}` ?

This is doc for default system variables in Github actions
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables

**Reply by simonjur** [reply]
yep I can confirm that runner name is there in JS through
```
process.env.RUNNER_NAME
```

and it looks like this on self-hosted runner:
```
...
  "GITHUB_ACTION_REF": "main",
  "RUNNER_OS": "Linux",
  "RUNNER_ARCH": "X64",
  "RUNNER_NAME": "github-actions-runner24-1",
  "RUNNER_ENVIRONMENT": "self-hosted",   <-------------- THIS
  "RUNNER_TOOL_CACHE": "/home/actions-runner-1/runner/_work/_tool",
...
```
 and like this on gh-hosted runner:
```
...
  "RUNNER_OS": "Linux",
  "RUNNER_ARCH": "X64",
  "RUNNER_NAME": "GitHub Actions 11",   <-------------- THIS
  "RUNNER_ENVIRONMENT": "github-hosted",
  "RUNNER_TEMP": "/home/runner/work/_temp",
...
```
